packages/driver/src/config/jquery.ts

```diff
@@ -1,11 +1,17 @@
-// @ts-nocheck
-
-import $ from 'jquery'
+import JQuery from 'jquery'
 import _ from 'lodash'
 
 import { scrollTo } from './jquery.scrollto'
 import $dom from '../dom'
 
+// Add missing types.
+interface ExtendedJQueryStatic extends JQueryStatic {
```

Comment:
Won't this caused typescript errors for individuals using JQuery methods other than find and expr?

Reply:
nvm. this isn't the types provided via the cli! sorry aboutt that